docs/source/guide/tutorials/advanced/semi_sl.rst

```diff
@@ -2,6 +2,141 @@
+
+
+2. To start training we need to call ``otx train``
+command in our worspace:
+
+.. code-block::
+
+  (otx) ...$ otx train
+
+In the train log, you can check that the train type is set to **SEMISUPERVISED** and related configurations are properly loaded as following:
+
+.. code-block::
+
+  ...
+  2023-02-22 06:21:54,492 | INFO : called _init_recipe()
+  2023-02-22 06:21:54,492 | INFO : train type = SEMISUPERVISED
+  2023-02-22 06:21:54,492 | INFO : train type = SEMISUPERVISED - loading training_extensions/otx/recipes/stages/classification/semisl.yaml
+  2023-02-22 06:21:54,500 | INFO : Replacing runner from EpochRunnerWithCancel to EpochRunnerWithCancel.
+  2023-02-22 06:21:54,503 | INFO : initialized recipe = training_extensions/otx/recipes/stages/classification/semisl.yaml
+  ...
+
+
+After training ends, a trained model is saved in ``models`` sub-directory in workspace ``otx-workspace-CLASSIFICATION``.
+
+
+3. In the same manner with :doc:`the normal validation <../base/how_to_train/classification#validation>`, 
+we can evaluate the trained model in ``models`` folder with auto-splitted validation dataset 
+and save results to ``performance.json`` by the following command:
+
+.. code-block::
+
+  (otx) ...$ otx eval --test-data-roots splitted_dataset/val \
+                      --load-weights models/weights.pth \
+                      --save-performance performance.json
+
+***************************
+Enable via ``otx train``
+***************************
+
+1. To enable semi-supervised learning directly via ``otx train``, we need to add arguments ``--unlabeled-data-roots`` and ``--algo_backend.train_type`` 
+which is one of template-specific parameters (The details are provided in :doc:`quick start guide <../../get_started/quick_start_guide/installation>`.)
+
+.. code-block::
+
+  (otx) ...$ otx train otx/algorithms/classification/configs/mobilenet_v3_large_1_cls_incr/template.yaml \
+                      --train-data-roots data/flower_photos \
+                      --unlabeled-data-roots tests/assets/imagenet_dataset \
+                      params --algo_backend.train_type SEMISUPERVISED
+
+In the train log, you can check that the train type is set to **SEMISUPERVISED** and related configurations are properly loaded as following:
+
+.. code-block::
+
+  ...
+  2023-02-22 06:21:54,492 | INFO : called _init_recipe()
+  2023-02-22 06:21:54,492 | INFO : train type = SEMISUPERVISED
+  2023-02-22 06:21:54,492 | INFO : train type = SEMISUPERVISED - loading training_extensions/otx/recipes/stages/classification/semisl.yaml
+  2023-02-22 06:21:54,500 | INFO : Replacing runner from EpochRunnerWithCancel to EpochRunnerWithCancel.
+  2023-02-22 06:21:54,503 | INFO : initialized recipe = training_extensions/otx/recipes/stages/classification/semisl.yaml
+  ...
+
+
+After training ends, a trained model is saved in ``otx-workspace-CLASSIFICATION/models`` since OTX generates a workspace named ``otx-workspace-CLASSIFICATION`` in default.
```

Comment:
```suggestion
After training ends, a trained model is saved in ``./models`` by default.
```
Just to generalize all cases

Reply:
Well it is true though. OTX generates a workspace named otx-workspace-{task name}. I don't understand what you want to generalize.

Reply:
I think I got your point. Your intention was that eval should be in other section so it should be explained globally. Did I get it right? I revised it like that. Could you see again?

Reply:
I mean, that we change our directory to "otx-workspace-CLASSIFICATION" above and now it is ``./models``. Also, if the user will run otx train without otx build it will be saved just in ``./models`` by default as far as I remeber. So, I just wanted to omit "otx-workspace-CLASSIFICATION" in folder name since we are inside that folder. Anyway, it is just minor comment

Reply:
I saw the changes and it looks good to me now